BfA/Underrot/Trash.lua

```diff
@@ -326,6 +328,24 @@ do
 	end
 end
 
+
+function mod:WickedEmbraceApplied(args)
+	if self:Me(args.destGUID) or self:Healer() or self:Dispeller("magic") then
+		local healthPercent = (UnitHealth(args.destName) / UnitHealthMax(args.destName)) * 100
+		if healthPercent <= 50 then
+			-- If the player's health is below or equal to 50%, play the low health warning
+			self:TargetMessage(args.spellId, "red", args.destName)
+			self:PlaySound(args.spellId, "alert")
```

Comment:
Missing the `, nil, args.destName` params on this :PlaySound, this is needed so the sound can be skipped if a player selects "only on me" in the options for this ability.